scriptmodules/libretrocores/lr-flycast.sh

```diff
@@ -31,7 +31,13 @@ function build_lr-flycast() {
     make clean
     if isPlatform "rpi"; then
         # MAKEFLAGS replace removes any distcc from path, as it segfaults with cross compiler and lto
-        MAKEFLAGS="${MAKEFLAGS/\/usr\/lib\/distcc:/}" make platform=rpi
+        MAKEFLAGS="${MAKEFLAGS/\/usr\/lib\/distcc:/}"
```

Comment:
This won't work as expected because the variable `MAKEFLAGS` is not exported, i.e. `make` will not see it if not given as an invocatio-variable (like is done before this PR).

You either do:

    VARIABLE=foo command

or:

    export VARIABLE=foo
    command

However, I think a better solution is to adopt the following idiom (used in other scriptmodules):

    local -a params=()
    if isPlatform "rpi4"; then
        params+=("platform=rpi4")
    else
        params+=("platform=rpi")
    fi
    MAKEFLAGS="${MAKEFLAGS/\/usr\/lib\/distcc:/}" make "${params[@]}"

Reply:
thanks for the correction! changes made and seem to test fine (although I don't have a pi4 so I'm presuming the new compile options are good since they were tested upstream). Squashed and good to go